### ISSUE TITLE
There doesn't seem to be a way to pass "retention-days" to CFLite jobs uploading artifacts

### ISSUE THREAD

**Issue by evverx** [google/clusterfuzzlite#81]
It would be great if by analogy with https://github.com/actions/upload-artifact#retention-period it was possible to pass something like `retention-days: 3` to CFLite jobs uploading artifacts. It would be even better if CFLite could automatically delete artifacts that should no longer be kept (like the "latest" builds generated a week ago) but I'm not sure how exactly it should work.

**Comment by jonathanmetzman** [comment]
Maybe we can set different retention periods for different artifacts.

**Comment by evverx** [comment]
I agree that CFLite shouldn't delete artifacts left and right :-) but I think that it should be configurable because ideally the retention policy should depend on how often projects are updated and I'm not sure CFLite can (or should) figure it out automatically.
